plugins/logs/service_events

```diff
@@ -208,8 +209,8 @@ done 3< <(echo "$vars")
 n=0
 allvars=0
 while [ $n -lt "${#reqvars[@]}" ]; do
-    !((allvars+=$(( 2 ** $n ))))
-    !((n++))
+    allvars=$(( allvars + $(( 2 ** $n )) ))
```

Comment:
I think, you can cut this down to `allvars=$((allvars + 2 ** n))`?

Reply:
Haha, for stuff like this, feel free to just merge it and then fix to your liking. I'm attached to the functionality, not the implementation, so don't worry about offending me by making unsolicited changes. But I think if we go back and forth like this we'll end up with 100 commits just to add 20 lines of code, and we'll both have lost our Sunday ;).

Reply:
(That's not to say I'm not learning from this, because I am, and I do value that, even in this relatively trivial example. But I can also learn from what you change post-merge, especially if you comment on why you changed it then.)

Reply:
Good point :)

Thus I may conclude: I feel fine the current set of changes, that you are proposing. Thus if you feel fine, too, I will merge it. Have a good remaining Sunday!

Reply:
Great, sounds good! And thanks again for working with me. I've already learned 5 concrete new bash/gnutils things just from these two PRs :D.